File: perl/fizzbuzz.pl
#!/usr/bin/perl

for my $i (1..100) {
    if ($i % 3 == 0 && print "Fizz") {
        if ($i % 5 == 0 && print "Buzz") {}
    } else {
        if ($i % 5 == 0 && print "Buzz") {}
        elsif (print $i) {}
    }
    
    if (print "\n") {}
}

File: java/FizzBuzz.java
import java.util.stream.IntStream;
import java.util.stream.Collectors;

public class FizzBuzz {
  public static void main(String[] args) {
    for (int n : IntStream.range(0, 100).boxed().collect(Collectors.toList())) {
      if (n % 3 == 0 && n % 5 == 0 && System.out.printf("FizzBuzz") != null) {}
      else if (n % 3 == 0 && System.out.printf("Fizz") != null) {}
      else if (n % 5 == 0 && System.out.printf("Buzz") != null) {}
      else if (System.out.printf(String.valueOf(n)) != null) {}
      
      if (System.out.printf("\n") != null) {}
    }                                       
  }
}
